EIPS/eip-4445.md

```diff
@@ -0,0 +1,195 @@
+---
+eip: 4445
```

Comment:
```suggestion
eip: 4400
```
Where did you get `4445` from?

Reply:
I thought that since 4400 has already passed, my PR should be updated to the next "available", but it seems that I was wrong. I've changed it back to the original one